velox/exec/tests/utils/ReferenceQueryRunner.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/exec/tests/utils/QueryAssertions.h"
```

Comment:
Consider moving this include to the `DuckQueryRunner.cpp` file.
You must include forward-declare or use only the required std headers.

Reply:
The reason is that when you build the PrestoQueryRunner later, this `include` will enforce a dependency on DuckDB.

Reply:
@majetideepak These are good points. Thanks. Updated.